server/services/alertsPageService.test.ts

```diff
@@ -2,21 +2,19 @@ import { Prisoner } from '../interfaces/prisoner'
 import AlertsPageService from './alertsPageService'
 import { inmateDetailMock } from '../data/localMockData/inmateDetailMock'
 import { pagedActiveAlertsMock, pagedInactiveAlertsMock } from '../data/localMockData/pagedAlertsMock'
-import PrisonApiRestClient from '../data/prisonApiClient'
-import { PrisonApiClient } from '../data/interfaces/prisonApiClient'
 import { PagedListQueryParams } from '../interfaces/prisonApi/pagedList'
 
+jest.mock('../data/prisonApiClient')
+
 describe('Alerts Page', () => {
-  let prisonApiClient: PrisonApiClient
   let getInmateDetailsSpy: jest.SpyInstance
   let getAlertsSpy: jest.SpyInstance
   let prisonerData: Prisoner
   let alertsPageService: AlertsPageService
 
   beforeEach(() => {
-    prisonApiClient = new PrisonApiRestClient(null)
     prisonerData = { bookingId: 123456, firstName: 'JOHN', lastName: 'SMITH' } as Prisoner
-    alertsPageService = new AlertsPageService(prisonApiClient)
+    alertsPageService = new AlertsPageService(null)
```

Comment:
We could do with making a generic prison api mock (we have one made manually in 3 of the other services) - this way you can avoid this and make some of the setup a bit cleaner, e.g.: 

where you have the `getInmateDetailsSpy` below you can change it to:
`prisonApiClient.getInmateDetail = jest.fn(async () => ({return: 'value'})`